README.md

```diff
@@ -1 +1,41 @@
-# ES6_Awesome_books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# ES6_Awesome_books
+
+A list of books project
+
+## Built With
+
+- HTML and Javascript
+
+
+## Getting Started
```

Comment:
- [ ] It is very important to add some set of instructions stating how users can run this project on their local machines. Such as cloning and switching to the expected branch then running `npm i`